src/api/app/jobs/staging_project_accept_job.rb

```diff
@@ -0,0 +1,10 @@
+class StagingProjectAcceptJob < ApplicationJob
+  queue_as :quick
+
+  def perform(project_id, user_login)
```

Comment:
why the project by id and the user by login?

Reply:
I just picked it since it's an identifier for users. There isn't any special reason why I picked this and not the id.